pytorch_lightning/trainer/trainer.py

```diff
@@ -953,8 +953,11 @@ def tune(
 
     def _restore_modules_and_callbacks(self) -> None:
         # restore modules after setup
-        if self.state.fn == TrainerFn.FITTING:
-            self.checkpoint_connector.resume_start()
+        self.checkpoint_connector.resume_start()
```

Comment:
then can we move this outside to where _restore_modules_and_callbacks() is called?

Reply:
yeah make sense.. done